assets/locales/en/main.json

```diff
@@ -64,39 +64,42 @@
     "slow-connection-2": "You might want to reload the page and try again. Please note you might lose the latest change to the street. Sorry!",
     "no-connection": "Streetmix is having trouble connecting to the Internet."
   },
-	"errors": {
-		"buttons": {
-			"home-button": "Go to the homepage",
-			"sign-in-button": "Sign in again",
-			"reload-button": "Reload the page",
-			"try-again-button": "Try again",
-			"general-help-buttons": "<a target='_blank' href='mailto:hello@streetmix.net'>email</a> or <a target='_blank' href='https://twitter.com/intent/tweet?text=@streetmix'>Twitter</a>. <br /> <button id = 'error-home'onClick = {goHome} > Go to the homepage < /button>"
+	"error": {
+		"button": {
+			"home": "Go to the homepage",
+			"sign-in": "Sign in again",
+			"reload": "Reload the page",
+			"try-again": "Try again",
+      "view-example": "View an example street"
 		},
 		"page-not-found-title": "Page not found.",
-		"page-not-found-description": "Oh, boy. There is no page with this address!<br />{homeButton}",
+		"page-not-found-description": "Oh, boy. There is no page with this address!",
 		"street-not-found-title": "Street not found.",
-		"street-not-found-description": "Oh, boy. There is no street with this link!<br />{homeButton}",
-		"street-not-found-but-user-description": "There is no street with this link! But you can look at other streets by {linkToUser(this.props.street)} <br />{homeButton}",
+		"street-not-found-description": "Oh, boy. There is no street with this link!",
+		"street-not-found-but-user-description": "There is no street with this link! But you can look at other streets by {linkToUser(this.props.street)}",
```

Comment:
It looks like that the value inside the placeholder was originally a JavaScript function, which we won't need here. I would simplify this to `{user}`.